sbnobj/SBND/Blip/CMakeLists.txt

```
@@ -0,0 +1,11 @@
+cet_make_library( LIBRARY_NAME sbndobj_BlipDataTypes
```

```suggestion
cet_make_library(
```
Library names are very hard to guess. A verbose but effective strategy is to leave it the default, which makes it easily predictable. And the most used convention guarantees at least the first element to be the repository the library belongs to ("`sbndobj`"?). I _request_ it does not start with `sbndobj`, which is misleading, and strongly _recommend_ to leave this library name the default:
 * in `CMakeLists.txt` link lists, people will have to write the annoyingly long `sbnobj::SBND_Blip` (could actually be worse);
 * when you `#include "sbnobj/SBND/Blip/BlipDataTypes.h"`, you _know_ that the library you need to add follows the path of the header (other example:
     ```cpp
     #include "lardataobj/RecoBase/Hit.h"
     #include "nusimdata/SimulationBase/MCParticle.h"
     ```
   links to `lardataobj::RecoBase` and `nusimdata::SimulationBase`. Presto.)

---

Updated the library name here and in relevant include locations.